lib/git_crc32.mli

```diff
@@ -0,0 +1,11 @@
+val cstruct: ?crc:int32 -> Cstruct.t -> int32
```

Comment:
Do you mind adding a copyright header in the mli too and removing the extra lines at the end?